packages/manager/modules/pci/src/projects/new/payment/routing.js

```diff
@@ -38,8 +50,24 @@ export default /* @ngInject */ ($stateProvider) => {
       paymentStatus: /* @ngInject */ ($transition$) =>
         $transition$.params().paymentStatus,
 
-      defaultPaymentMethod: /* @ngInject */ (ovhPaymentMethod) =>
-        ovhPaymentMethod.getDefaultPaymentMethod(),
+      paymentMethods: /* @ngInject */ (ovhPaymentMethod) =>
+        ovhPaymentMethod.getAllPaymentMethods(),
```

Comment:
I don't know how much impact there is to get all methods but maybe we should only do so if we know it's needed (like when there is no default payment method for instance)

Reply:
To know if there is a default payment method, we have to get all payment methods. So here I don't use `getDefaultPaymentMethod` directly because I need to list payment methods. So I get all payment methods to first know if there is or not a default payment method and if not display a list of valid payment methods. But in all the cases we have to get all payment methods to have their details...